### DISCUSSION TITLE
How do detect which entity saved in 4.16.0 ?

### DISCUSSION THREAD

**Discussion by lotarbo** [statamic/cms#8588] [Q&A] [answered]
Hello, i have multisite with 20 sites.
i listen to the event EntrySaved to trigger pipelines
After this update
https://github.com/statamic/cms/pull/8505

Saving origin entity triggers 20 events -> 20 pipelines
Right now i cant detect in which site entity save, because events started from last site.

In my scenario, i need to know in which site entity updated in admin panel. How i can do it after upgrade?

**Comment by duncanmcclean** [comment]
This sounds like an oversight from that PR - would you be able to open a bug report?

**Comment by jasonvarga** [comment, marked as answer]
This is being addressed in #8605.

You'll be able to do `$event->isInitial()` to check if it was the first.

**Reply by lotarbo** [reply]
@jasonvarga  thank you
